app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -128,8 +128,11 @@ def parse_build_pod(build_pod)
           :start_timestamp               => status[:startTimestamp],
           :output_docker_image_reference => status[:outputDockerImageReference],
         )
-        new_result[:build_config] = @data_index.fetch_path(path_for_entity("build_config"), :by_name,
-                                                           build_pod.status.config.try(:name))
+        bc_name = build_pod.status.config.try(:name)
+        bc_namespace = build_pod.status.config.try(:namespace)
+        new_result[:build_config] = @data_index.fetch_path(path_for_entity("build_config"),
+                                                           :by_namespace_and_name,
+                                                           bc_namespace, bc_name)
```

Comment:
@enoodle probably we don't need this indentation (you may even keep a single line).